packages/contentstack-export-to-csv/src/commands/cm/export-to-csv.js

```diff
@@ -117,6 +118,11 @@ class ExportToCsvCommand extends Command {
             let contentTypes = [];
 
             if (managementTokenAlias) {
+              const listOfTokens = configHandler.get('tokens');
+              const checkManagementTokenValidity = await isManagementTokenValid(stackAPIKey,listOfTokens[managementTokenAlias].token);
+              if(!checkManagementTokenValidity?.valid) {
```

Comment:
@cs-raj , its better to destructure valid and message values from "checkManagementTokenValidity" object and have a default values for it. Instead of using ? multiple places  in this code

Reply:
I would need that as an object as when we fail to validate the management token validity we are not passing the valid property